helm/chaos-mesh/templates/chaos-daemon-daemonset.yaml

```diff
@@ -140,3 +136,15 @@ spec:
           hostPath:
             path: /usr/src
 {{- end }}
+      {{- with .Values.chaosDaemon.nodeSelector }}
+      nodeSelector:
+{{ toYaml . | indent 8 }}
+      {{- end }}
+      {{- with .Values.chaosDaemon.affinity }}
+      affinity:
```

Comment:
I think daemonset chaos-daemon do not need affinity/anti-affinity. It will only runs one instance on one node.